pkg/common/api/middleware/names.go

```diff
@@ -19,6 +19,7 @@ const (
 	EnvoySDSv3ServiceShortName        = "SDS.v3"
 	HealthServiceName                 = "grpc.health.v1.Health"
 	HealthServiceShortName            = "Health"
+	LoggerServiceName                 = "logger.v1.Logger"
```

Comment:
unused?

Reply:
From in-chat conversations, apparently this needs more code (not less) to complete the name replacing service.

Fixed in next push